data/data-pipeline/data_pipeline/etl/score/tests/test_score_post.py

```diff
@@ -89,21 +88,21 @@ def test_create_tile_data(etl, score_data_expected, tile_data_expected):
     )
 
 
-def test_create_downloadable_data(
-    etl, score_data_expected, downloadable_data_expected
-):
-    output_downloadable_df_actual = etl._create_downloadable_data(
-        score_data_expected
-    )
-    pdt.assert_frame_equal(
-        output_downloadable_df_actual,
-        downloadable_data_expected,
-    )
+# def test_create_downloadable_data(
+#     etl, score_data_expected, downloadable_data_expected
+# ):
+#     output_downloadable_df_actual = etl._create_downloadable_data(
+#         score_data_expected
+#     )
+#     pdt.assert_frame_equal(
+#         output_downloadable_df_actual,
+#         downloadable_data_expected,
+#     )
```

Comment:
Note: These tests will be refactored to something else at some point, it doesn't make sense to have them when content/product can edit the YAML and expect them to produce a Python pickle for the tests.